twentytwenty-blocks/readme.txt

```diff
@@ -0,0 +1,98 @@
+=== Twenty Twenty ===
+Contributors: the WordPress team
+Tested up to: 5.3
+Stable tag: 1.1
+License: GPLv2 or later
+License URI: http://www.gnu.org/licenses/gpl-2.0.html
+
+Default theme for 2020.
```

Comment:
This should be updated a bit so that it's clear it's an experiment and not the actual 2020 theme. Can also include instructions for installing it so that the starter content kicks in. You can check out what I did for this in #4:

https://github.com/WordPress/theme-experiments/blob/16f681b6beaa27efef4aa028e391b424f13339ab/twentynineteen-blocks/readme.txt